index.html

```diff
@@ -37,7 +37,7 @@
   <body>
     <section id='abstract'>
       <p>
-        The new input event type introduced in this document provide web developers a way to listen to "click" action of non-primary buttons and also prevent the default behaviour on those events like opening a new tab when middle clicking on a link. This new input type is needed because the <a href="https://www.w3.org/TR/DOM-Level-3-Events/#event-type-click">click</a> event is restricted to primary button only.
+        The new input event type introduced in this document provides web developers a way to listen to the "click" action of non-primary buttons, and potentially prevent their default action (eg. opening a new tab when middle clicking on a link). This new input type is needed because the <a href="https://www.w3.org/TR/DOM-Level-3-Events/#event-type-click">click</a> event is restricted to primary button only.
```

Comment:
Regarding the word "default action" vs "default behavior", I was not sure which one was better. Particularly since we used "a way to listen to the "click" action" in the previous sentence. So if we use the "action" again the second one does mean something else. Wouldn't that be confusing?